### PR TITLE
COCICCS-5083:T-Vault (AAD): Show the list of AAD Service Principals in owner page

### PR DESCRIPTION
**Pull request by msivatmob**

### CODE REVIEW

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/AzureServicePrinicipalAccountsService.java [hunk 1]
```diff
@@ -1355,5 +1355,47 @@ private String dateConversion(Long createdEpoch) {
...
+			if (policy.startsWith("o_azuresvcacc")) {
+				onboardedlist.add(policy.substring(14));
+			}
+		}
+		response = new Response();
+		response.setHttpstatus(HttpStatus.OK);
+		response.setSuccess(true);
+		response.setResponse("{\"keys\":" + JSONUtil.getJSON(onboardedlist) + "}");
+		
+		if (HttpStatus.OK.equals(response.getHttpstatus())) {
```
Comment (sarathmohan4) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/AzureServicePrinicipalAccountsService.java:1387]: This check is always true, is it required?

File: tvaultapi/src/main/resources/application.properties [hunk 2]
```diff
@@ -7,17 +7,17 @@ vault.secret.key.whitelistedchars=[a-z0-9_]+
...
-ldap.userdn:
-ldap.username:
-ldap.password:
-
-ad.url=
-ad.base:
-ad.userdn:
-ad.username:
-ad.password:
+ldap.url=
```
Comment (sarathmohan4) [inline tvaultapi/src/main/resources/application.properties:10]: additional spaces,

File: tvaultapi/src/main/resources/application.properties [hunk 3]
```diff
@@ -34,7 +34,7 @@ selfservice.ssfilelocation=/opt/tvault/hcorp
 spring.mail.host=email.smtp
 spring.mail.port=123
 
-workload.endpoint=
+workload.endpoint=
```
Comment (sarathmohan4) [inline tvaultapi/src/main/resources/application.properties:37]: additional space

File: tvaultapi/src/main/resources/application.properties [hunk 4]
```diff
@@ -463,8 +463,8 @@ CertificateController.getCertificateDetails.value= To get certificate details.
 CertificateController.getCertificateDetails.notes= To get certificate details.
 
 
-sslcertmanager.domain=
-sslcertmanager.username=
+sslcertmanager.domain=
```
Comment (sarathmohan4) [inline tvaultapi/src/main/resources/application.properties:466]: additional space

File: tvaultui/src/app/Features/admin/admin.controller.js [hunk 5]
```diff
@@ -959,6 +992,17 @@
             }
                return iampagesShown < ($scope.numOfIamSvcaccs / iampageSize);
```
Comment (sarathmohan4) [inline tvaultui/src/app/Features/admin/admin.controller.js:993]: iampagesShown , using same variable might impact padination in IAM lisiting

**Review by sarathmohan4 — CHANGES_REQUESTED** [review]

File: tvaultapi/src/main/resources/application.properties (hunk 3 above)
Comment (msivatmob) [reply tvaultapi/src/main/resources/application.properties:35]: resolved

File: tvaultui/src/app/Features/admin/admin.controller.js (hunk 5 above)
Comment (msivatmob) [reply tvaultui/src/app/Features/admin/admin.controller.js:993]: changed the variable name to azurepagesshown.

File: tvaultui/src/app/Features/admin/admin.jade [hunk 6]
```diff
@@ -849,6 +849,44 @@ script(type='text/ng-template', id='manageSecretId.html')
                                                     button.show-more-btn VIEW MORE
                                                     span
                                                         img(src="assets/icons/front-arrow.svg")
+                    md-tab(label="Azure Service Principals" md-on-select="tabChangeForAdmin()" ng-if="enableAzureSvcacc")
+                        md-content(class="md-padding md-content-wrapper page-content-wrapper md-approle-management" ng-init="displayGrid")
+                            .new-container.page-content-container
+                                .md-content-inner-wrapper
+                                    div.svcnotes T-Vault can be used to manage to read and rotate the secrets of Azure service accounts. In order to self-service your Azure service accounts there is a three-step process to onboard the account into T-Vault.
+                                      span.svc-collapse-link(ng-click="collapseNoteIAM()") >>
```
Comment (sarathmohan4) [inline tvaultui/src/app/Features/admin/admin.jade:857]: collapseNoteIAM() and isCollapsedIAM is already used in IAM tab, might need to use a new function and variable here

**Review by sarathmohan4 — CHANGES_REQUESTED** [review]

File: tvaultui/src/app/Features/admin/admin.jade (hunk 6 above)
Comment (msivatmob) [reply tvaultui/src/app/Features/admin/admin.jade]: changed to collapseNoteAzure() and CollapsedAzure.

**Review by alexrajan3 — COMMENTED** [review]
Approved

**Pull request merged**